examples/mintDaiAndSwap.ts

```diff
@@ -0,0 +1,179 @@
+import { Order, OrderBalance, OrderKind } from '@cowprotocol/contracts';
+import { CowShedSdk, ICall } from '../ts';
+import {
+  ABI_CODER,
+  COW,
+  DAI,
+  DAI_JOIN,
+  ETH_A_JOIN,
+  VAT,
+  VAULT_RELAYER,
+  WETH,
+  approveToken,
+  createOrder,
+  fnCalldata,
+  getIlk,
+  joinEth,
+  resolveName,
+  settleOrder,
+  vatHope,
+  withAnvilProvider,
+  wrapEther,
+} from './common';
+import { ethers } from 'ethers_v6';
+
+const mintDaiAndSwap: Parameters<typeof withAnvilProvider>[0] = async (
+  provider: ethers.JsonRpcProvider,
+  signers: ethers.Wallet[],
+  { factory, implementation, proxyInitCode }
+) => {
+  // initialize the sdk
+  const shedSdk = new CowShedSdk({
+    factoryAddress: factory,
+    implementationAddress: implementation,
+    proxyCreationCode: proxyInitCode,
+    chainId: 1,
+  });
+
+  const user = signers[1];
+  // collateral to deposit
+  const collateral = ethers.parseEther('10');
+  // debt to take out
+  const daiDebt = ethers.parseEther('8000');
+
+  const proxyAddress = shedSdk.computeProxyAddress(user.address);
+  console.log(
+    'Computed proxy address for user',
+    user.address,
+    'is',
+    proxyAddress
+  );
+  await wrapEther(provider, user.address, collateral);
+  // allow proxy to take actions on behalf of the user
+  await vatHope(provider, user.address, proxyAddress);
+  // approve and deposit WETH into WETH Join
+  await approveToken(provider, WETH, user.address, ETH_A_JOIN, collateral);
+  await joinEth(provider, user.address, collateral);
+
+  const validTo = Math.floor(new Date().getTime() / 1000) + 7200;
+  const buyAmount = ethers.parseEther('100');
+
+  // create a sell order for DAI to COW
+  const order: Order = {
+    sellToken: DAI,
+    buyToken: COW,
+    receiver: proxyAddress,
```

Comment:
The receiver should be the `user`, no? The proxy has done the CDP manipulation for the `user`, and shouldn't be the recipient (there are no further actions, so the recipient could be the `user` again?